packages/cashier/src/Components/payment-agent-list.jsx

```diff
@@ -65,112 +74,104 @@ const PaymentAgentList = ({
-                        weight='bold'
-                        color='prominent'
-                        className='cashier__header payment-agent__list-header'
-                    >
-                        <Localize i18n_default_text='Payment agents' />
-                    </Text>
-                    <div className='payment-agent__list-line' />
-                    {is_loading ? (
-                        <Loading className='payment-agent__loader' />
-                    ) : (
-                        <React.Fragment>
-                            {supported_banks.length > 1 && (
-                                <div className='payment-agent__list-selector'>
-                                    <DesktopWrapper>
-                                        <Dropdown
-                                            id='payment_methods'
-                                            className='payment-agent__drop-down payment-agent__filter'
-                                            classNameDisplay='cashier__drop-down-display payment-agent__filter-display'
-                                            classNameDisplaySpan='cashier__drop-down-display-span'
-                                            classNameItems='cashier__drop-down-items'
-                                            list={list_with_default}
-                                            name='payment_methods'
-                                            value={selected_bank}
-                                            onChange={onChangePaymentMethod}
-                                        />
-                                    </DesktopWrapper>
-                                    <MobileWrapper>
-                                        <SelectNative
-                                            placeholder={localize('Please select')}
-                                            name='payment_methods'
-                                            list_items={supported_banks}
-                                            value={selected_bank}
-                                            label={
-                                                selected_bank === 0 ? localize('All payment agents') : localize('Type')
-                                            }
-                                            onChange={e =>
-                                                onChangePaymentMethod({
-                                                    target: {
-                                                        name: 'payment_methods',
-                                                        value: e.target.value.toLowerCase(),
-                                                    },
-                                                })
-                                            }
-                                            use_text={false}
+                        <Tabs
+                            active_index={verification_code || is_payment_agent_withdraw ? 1 : 0}
```

Comment:
Could you define a state for `active_tab_index` and use that here and in the condition of `onTabItemClick` instead of `index`.

Reply:
noted, done